modules/osdk-updating-projects.adoc

```diff
@@ -0,0 +1,353 @@
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+
+:osdk_ver: 1.22.0
+:osdk_ver_n1: 1.16.0
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed.
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}.
+
+.Procedure
+
+. Make the following changes to the `config/default/manager_auth_proxy_patch.yaml` file: 
++
+[source,yaml]
+----
+...
+spec:
+  template:
+    spec:
+      containers:
+      - name: kube-rbac-proxy
+        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.11 <1>
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://127.0.0.1:8080/"
+        - "--logtostderr=true"
+        - "--v=0" <2>
+...
+resources:
+  limits:
+    cpu: 500m
+    memory: 128Mi
+  requests:
+    cpu: 5m
+    memory: 64Mi <3>
+----
+<1> Update the tag version from `v4.10` to `v4.11.`.
+<2> Reduce the debugging log level from `"--v10"` to `"--v=0"`.
+<3> Add resource requests and limits.
+
+. Make the following changes to your `Makefile`:
+
+.. Enable support for image digests by adding the following environment variables:
++
+.Old `Makefile`
+[source,terminal]
+----
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+...
+----
++
+.New `Makefile`
+[source,terminal]
+----
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+
+# BUNDLE_GEN_FLAGS are the flags passed to the operator-sdk generate bundle command
+BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+
+# USE_IMAGE_DIGESTS defines if images are resolved via tags or digests
+# You can enable this value if you would like to use SHA Based Digests
+# To enable set flag to true
+USE_IMAGE_DIGESTS ?= false
+ifeq ($(USE_IMAGE_DIGESTS), true)
+	BUNDLE_GEN_FLAGS += --use-image-digests
+endif
+----
+
+.. Replace the bundle target with the `BUNDLE_GEN_FLAGS` environment variable:
++
+.Old `Makefile`
+[source,terminal]
+----
+$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+----
++
+.New `Makefile`
+[source,terminal]
+----
+$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
+---- 
+
+.. Update `opm` to version 1.23.0:
```

Comment:
Again, could be because I'm not familiar with your OSDK stuff, but I don't know what I'm supposed to do with this. Am I editing the makefile again, or is this somewhere else?